core/src/com/po/fuck/model/GameObject.java

```diff
@@ -1,25 +1,21 @@
-package com.po.fuck;
+package com.po.fuck.model;
 
-import com.badlogic.gdx.graphics.g2d.Sprite;
 import com.badlogic.gdx.math.Polygon;
 import com.badlogic.gdx.math.Vector2;
-import com.po.fuck.collision.Collidable;
-import com.po.fuck.updates.Drawable;
-import com.po.fuck.lifetime.Destructable;
+import com.po.fuck.model.collision.Collidable;
+import com.po.fuck.model.lifetime.Destructable;
 
-public class GameObject implements Drawable, Collidable, Destructable {
+public class GameObject implements Drawable, Collidable, Destructable, Updatable {
```

Comment:
Isn't it an overkill ?

Reply:
Do you think so?

Reply:
Yes, I have explained why in the total(IDK how is it named) comment.